modules/SerialCommHub/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#
+# AUTO GENERATED - MARKED REGIONS WILL BE KEPT
+# template version 3
+#
+
+# module setup:
+#   - ${MODULE_NAME}: module name
+ev_setup_cpp_module()
+
+# ev@bcc62523-e22b-41d7-ba2f-825b493a3c97:v1
+# insert your custom targets and additional config variables here
+
+target_link_libraries(${MODULE_NAME}
+    PRIVATE
+    everest::modbus
+    date::date
```

Comment:
Is date needed here?

Reply:
probably that's a left over, will remove